python/deltalake/table.py

```diff
@@ -464,7 +469,7 @@ def update(
         >>> dt = DeltaTable("tmp")
         >>> dt.update(predicate="id = '5'",
         ...           updates = {
-        ...             "deleted": True,
+        ...             "deleted": 'True',
```

Comment:
hmm we should add a doctest CI job. 🤔 https://github.com/delta-io/delta-rs/issues/1783

Reply:
Yeah we should, then this bug would have been catched the first PR.